Makefile

```diff
@@ -1,9 +1,9 @@
 all: build test
 
-CGO_ENABLED=0
+export CGO_ENABLED=0
```

Comment:
Without the `export`, it doesn't actually set the env var, so `CGO` is still enabled for the final binary.